ci-scripts/setup/.env-example

```diff
@@ -0,0 +1,49 @@
+# Adobe Reactor SDK Integration Test Environment Variables
+# Copy this file to .env and fill in your actual values
+
+########## DEFAULT COMPANY (PREMIUM CDN DISABLED) ##########
+# A "Default Company" has Dynamic CDN disabled.
+# Required: Adobe I/O Client Credentials (for @adobe/auth-token)
+# Get these from Adobe Developer Console when you create your integration
+RSDK_ADOBE_DEFAULT_COMPANY_CLIENT_ID=your_client_id_here
+RSDK_ADOBE_DEFAULT_COMPANY_CLIENT_SECRET=your_client_secret_here
+#
+# Required: Your Adobe Organization ID
+# Format: ORG123456789@AdobeOrg
+RSDK_ADOBE_DEFAULT_COMPANY_ORG_ID=your_org_id_here
+#
+# Required: Your Reactor Company ID
+# Format: CO1234567890abcdef1234567890abcdef
+RSDK_ADOBE_REACTOR_DEFAULT_COMPANY_ID=your_company_id_here
+#
+########## DEFAULT COMPANY (PREMIUM CDN DISABLED) ##########
```

Comment:
It might be worth explaining here what is going on with the disabled enabled stuff.
Or maybe just removing `########## DEFAULT COMPANY (PREMIUM CDN DISABLED) ##########`

i was looking at 
`########## DEFAULT COMPANY (PREMIUM CDN DISABLED) ##########`
followed by
`########## PREMIUM COMPANY (PREMIUM CDN ENABLED) ##########`

And I was like what now. why is `########## DEFAULT COMPANY (PREMIUM CDN DISABLED) ##########` here?

Also The Premium CDN stuff is not required unless you have a premium cdn enabled company. Correct? Might be good to call that out.